From d79847e47cd42f67cf0a5504abc3e19d434990ff Mon Sep 17 00:00:00 2001
From: Thao Luong <thao.luong.uw@rvc.renesas.com>
Date: Wed, 27 May 2015 11:35:29 +0700
Subject: [PATCH 16/25] config.ac: Add config to disable to export dmabuf  file
  descriptors

This patch adds config to disable to export dmabuf file descriptors
even if the mmngrbuf library is detected.
When dmabuf file descriptors need to be disable, please add "--disable-dmabuf"
as configure option.

refs #69206
Signed-off-by: Thao Luong <thao.luong.uw@rvc.renesas.com>
---
 configure.ac | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/configure.ac b/configure.ac
index 404eace..e66efd1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -261,11 +261,21 @@ elif test x"$ac_cv_omx_target_struct_packing" != x"none"; then
   AC_DEFINE_UNQUOTED(GST_OMX_STRUCT_PACKING, $ac_cv_omx_target_struct_packing, [The struct packing used for OpenMAX structures])
 fi
 
+dnl check for dmabuf support
+AC_ARG_ENABLE([dmabuf],
+             [AS_HELP_STRING([--disable-dmabuf],
+                             [Disable dmabuf even if you have mmngrbuf library])],
+             [],
+             [enable_dmabuf=yes]
+)
+if test "x${enable_dmabuf}" = "xyes"; then
+
 dnl check mmngrbuf library to use dmabuf exporter
 AC_CHECK_LIB([mmngrbuf], [mmngr_export_start_in_user],
              [AC_DEFINE(HAVE_MMNGRBUF, 1, [Define if you have mmngrbuf library])
               GST_LIBS="$GST_LIBS -lmmngrbuf"
              ])
+fi
 
 dnl *** set variables based on configure arguments ***
 
-- 
2.4.0

